cmd/start.go

```diff
@@ -195,7 +196,7 @@ func runContainer(cmd *cobra.Command, args []string) {
 				if sizeBluestoreBlockToBytes == 0 {
 					log.Fatal("Wrong unit passed: ", getSize(flavor), ". Please refer to https://en.wikipedia.org/wiki/Byte.")
 				}
-				envs = append(envs, "BLUESTORE_BLOCK_SIZE="+string(sizeBluestoreBlockToBytes))
+				envs = append(envs, "BLUESTORE_BLOCK_SIZE="+fmt.Sprint(sizeBluestoreBlockToBytes))
```

Comment:
Please use `strconv.Itoa()` instead.

Reply:
Looks like `strconv.Itoa()` expects `int` while here we have `int64`:

```
./start.go:198:61: cannot use sizeBluestoreBlockToBytes (type int64) as type int in argument to strconv.Itoa
```

So I used `strconv.FormatInt` instead.